src/npc.cpp

```diff
@@ -1651,28 +1697,35 @@ int npc::value(const item &it)
                 ret += book.level * 3;
             }
         }
- }
+    }
 
-// TODO: Sometimes we want more than one tool?  Also we don't want EVERY tool.
- if (it.is_tool() && !has_amount(itype_id(it.type->id), 1)) {
-  ret += 8;
- }
+    // TODO: Sometimes we want more than one tool?  Also we don't want EVERY tool.
+    if( it.is_tool() && !has_amount( itype_id(it.type->id), 1) ) {
+        ret += 8;
+    }
 
-// TODO: Artifact hunting from relevant factions
-// ALSO TODO: Bionics hunting from relevant factions
- if (fac_has_job(FACJOB_DRUGS) && it.is_food() &&
-     (dynamic_cast<const it_comest*>(it.type))->addict >= 5)
-  ret += 10;
- if (fac_has_job(FACJOB_DOCTORS) && it.type->id >= "bandages" &&
-     it.type->id <= "prozac")
-  ret += 10;
- if (fac_has_value(FACVAL_BOOKS) && it.is_book())
-  ret += 14;
- if (fac_has_job(FACJOB_SCAVENGE)) { // Computed last for _reasons_.
-  ret += 6;
-  ret *= 1.3;
- }
- return ret;
+    // TODO: Artifact hunting from relevant factions
+    // ALSO TODO: Bionics hunting from relevant factions
+    if( fac_has_job(FACJOB_DRUGS) && it.is_food() &&
+        (dynamic_cast<const it_comest*>(it.type))->addict >= 5 ) {
+        ret += 10;
+    }
+
+    if( fac_has_job(FACJOB_DOCTORS) && it.type->id >= "bandages" &&
+        it.type->id <= "prozac") {
```

Comment:
This (`<=` and `>=`) was/is actually wrong. It's a remnant of simpler times, when item type ids were integers. Now it compares strings, most trivial things like "sheet" match, but "quikclot", "adderall", "xanax", "thorazine" etc. don't.

It should probably either inspect the item directly (check its properties / iuse functions), or use an item flag. The flag can be set by the author of the item to indicate that it's an medical item which is looked for by that faction.